common/src/main/java/io/seata/common/util/NetUtil.java

```diff
@@ -164,16 +164,14 @@ private static InetAddress getLocalAddress0() {
                     try {
                         NetworkInterface network = interfaces.nextElement();
                         Enumeration<InetAddress> addresses = network.getInetAddresses();
-                        if (addresses != null) {
```

Comment:
It is recommended to keep and shield external changes.

Reply:
Ok. I corrected it.